packages/tabs/src/useTabs.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useState } from 'react';
+import { useSelection } from '@zendeskgarden/container-selection';
+import { generateId } from '@zendeskgarden/container-utilities';
+
+function requiredArguments(arg, argStr, methodName) {
```

Comment:
We do this argument checking in a few hooks should it move to container-utilities?

Reply:
This seems low-level enough I don't think the abstraction would help much. If it get's heavy maybe we could move it over in the future?

Reply:
Yes, let's do it! Thanks.

Reply:
So you both voting for the other option? Should I move it over or not?

Reply:
I'm fine either way.

Reply:
ok will leave as is for now